### ISSUE TITLE
Nuget restore failing for UWP project VS 2019

### ISSUE THREAD

**Issue by VladislavAntonyuk** [appveyor/ci#2864]
MSBuild auto-detection: using msbuild version '16.0.360.33175' from 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\MSBuild\Current\bin'.
142
C:\projects\draw-go\Platform\DrawGo.UWP.MainApp\DrawGo.UWP.MainApp.csproj(195,11): error MSB4226: The imported project "C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\MSBuild\Microsoft\WindowsXaml\v16.0\Microsoft.Windows.UI.Xaml.CSharp.targets" was not found. Also, tried to find "Microsoft\WindowsXaml\v16.0\Microsoft.Windows.UI.Xaml.CSharp.targets" in the fallback search path(s) for $(MSBuildExtensionsPath) - "C:\Program Files (x86)\MSBuild" . These search paths are defined in "C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\MSBuild\Current\bin\msbuild.exe.Config". Confirm that the path in the <Import> declaration is correct, and that the file exists on disk in one of the search paths.
143
C:\projects\draw-go\Platform\DrawGo.UWP\DrawGo.UWP.wapproj(35,3): error MSB4019: The imported project "C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\MSBuild\Microsoft\DesktopBridge\Microsoft.DesktopBridge.props" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.
144
145
An error occurred when executing task 'NuGetRestore'.
146
147
----------------------------------------
![image](https://user-images.githubusercontent.com/33021114/52653801-694e8a00-2ef9-11e9-95c8-379e905f3c87.png)

Missed components:
![image](https://user-images.githubusercontent.com/33021114/52653911-9ac75580-2ef9-11e9-9f1e-aa3d7ca972a6.png)

**Comment by IlyaFinkelshteyn** [comment]
@VladislavAntonyuk can you confirm that if you RDP (with **blocking** RDP) at `install` stage and install those missed components manually, then unblock RDP and let build run, this error disappears?

**Comment by VladislavAntonyuk** [comment]
I have setup rdp on failure, so I opened solution and found that uwp projects are not loaded. I clicked install missed components and after installation uwp project loaded successfully

**Comment by VladislavAntonyuk** [comment]
Also make sure you have enabled Developer mode in Windows Settings.

Also updating to preview 3.0 fix the problem without installing missed components

**Comment by IlyaFinkelshteyn** [comment]
@VladislavAntonyuk try new `Visual Studio 2019` image.

**Comment by VladislavAntonyuk** [comment]
error NETSDK1045: The current .NET SDK does not support targeting .NET Core 3.0.  Either target .NET Core 2.2 or lower, or use a version of the .NET SDK that supports .NET Core 3.0. 

Please set in VS settings to allow to use .net core preview

**Comment by IlyaFinkelshteyn** [comment]
OK, thanks for the information. Will fix this in context of `Visual Studio 2019 Preview` image then.

**Comment by VladislavAntonyuk** [comment]
Please set this flag in VS 2019 image too. Microsoft plans to release .NET core 3 in September, so it would be nice to have green builds with stable version of Visual Studio

**Comment by VladislavAntonyuk** [comment]
Required to work with .net core 3
![image](https://user-images.githubusercontent.com/33021114/58264981-82cc6100-7d87-11e9-98e8-5b070629dc5d.png)
Required for UWP (build failed until disabled)
![image](https://user-images.githubusercontent.com/33021114/58265071-ad1e1e80-7d87-11e9-9fa6-3d7b885934d8.png)
Required to build Xamarin.Mac and Xamarin.GTK
![image](https://user-images.githubusercontent.com/33021114/58265193-e191da80-7d87-11e9-9715-fb87f03452f5.png)

**Comment by godrose** [comment]
Any ETA on this? @IlyaFinkelshteyn

**Comment by FeodorFitsner** [comment]
Have you tried building with `Visual Studio 2019` image? It has the latest VS 2019 and .NET Core 3.0.

**Comment by godrose** [comment]
@FeodorFitsner Now it works. Thanks.
